tokio/src/runtime/task/raw.rs

```diff
@@ -6,7 +6,7 @@ use std::ptr::NonNull;
 use std::task::{Poll, Waker};
 
 /// Raw task handle
-pub(super) struct RawTask {
+pub(crate) struct RawTask {
```

Comment:
Why is this made `pub(crate)`. It is used by `tokio::runtime::task::trace` which should be covered by `pub(super)`.